### ISSUE TITLE
Is this repo still maintained?

### ISSUE THREAD

**Issue by Wargazm** [aziz/PlainNotes#60]
I like the functionality this plugin provides but I am seeing some bugginess (for example, `**text**` is not bolding in my notes).  Is there any active development here?

Thanks

**Comment by aziz** [comment]
> Is this repo still maintained?
Maintained is a relative term, but yes I'm using this plugin everyday myself and I'm looking for ways to improve it in the future. It should help my own workflow or it should be a pull request. 

> `**text**` is bold 
It is shows as bold for me. Maybe you're using a font that does not have a bold type. 

>  Is there any active development here?
I've stopped working on customizing the markdown syntax for this plugin since the official markdown syntax from sublime text is under active development these days and a lot of features I added to the old `tmLanguage` format with a lot of pain and tears are now being added properly to the `sublime-syntax` version of the official syntax shipping with Sublime. 
So yes, I slowed down to see what we can when the latest markdown syntax is released
